observability-kit-starter-hilla/pom.xml

```
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>observability-kit</artifactId>
+        <version>2.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>observability-kit-starter-hilla</artifactId>
+
+    <properties>
+        <cvdl.name>vaadin-observability-kit</cvdl.name>
```

Let's use `hilla-observability-kit` as the product slug, following the same convention we used for SSO Kit:
```suggestion
        <cvdl.name>hilla-observability-kit</cvdl.name>
```